test/extended/deployments/util.go

```diff
@@ -469,6 +469,10 @@ func createFixture(oc *exutil.CLI, fixture string) (string, string, error) {
 	return resource, parts[1], nil
 }
 
+func DeploymentConfigFailureTrap(oc *exutil.CLI, name string, failed bool) {
+	failureTrap(oc.AsAdmin(), name, failed)
+}
```

Comment:
@mfojtik, @jim-minter pointed out that this needs to pass the namespace becase AsAdmin() loses the active namespace.  Can you do a followup?

or just don't use the AsAdmin() since the oc client being passed in should be good enough for dumping what this needs.